### ISSUE TITLE
Show color contrast ratios more prominently

### ISSUE THREAD

**Issue by toolness** [toolness/accessible-color-matrix#9]
Because the color matrix came out of the 18F branding guide, it was designed to show readers a *finished* palette; it wasn't made for designers who are in the process of *creating* a palette to make it usable.

So e.g. if a color combination is *really close* to being accessible, but not quite, we're not currently communicating that at all--we just gray out the box in the grid because it's not accessible.  Hovering over the square does show the exact color contrast, but it's not very discoverable, nor easy to see at a glance.

So it might be better, at least when the user is in the "Edit" mode, to show the actual contrast ratios more prominently.
